### PR TITLE
Review appendix

### LINKED ISSUE #74: Proposed changes
In the Appendix, revise the sentence "The first survey....following questions" as follows: 
"The first survey, shown in its original form in Figure 18, that focused on the use of messaging apps in class,
and which have been used for addressing RQ1, RQ2, and RQ3, includes the following questions:"

Then, later on in the Appendix, delete the whole paragraph given before the survey 2 items (this information to be given in the methods) and add instead the following sentence: 
"The second survey, shown in its original form in Figure 19, focusing on the kind of interaction media features that teachers
value the most and which have been used for addressing RQ4 includes the following questions:"

### PR DESCRIPTION
**Pull request by fbarranco**
Just a few edits reviewing the appendix. It closes #74

### CODE REVIEW

File: edubots_surveys.Rnw [hunk 1]
```diff
@@ -1731,7 +1731,7 @@ All the authors have contributed to the study presented in this manuscript. (Res
 %
 % PAC has revised the paper and researched the state of the art, as well as contributed to the survey design.
 % AMM has revised the paper and researched the state of the art, as well as contributed to the survey design.
-% FB has made the analysis and processed data and contributed to the survey design.
+% FB has contributed to the survey design, made the analysis, processed data, and reviewed the writing of the paper.
```
Comment (JJ) [inline edubots_surveys.Rnw]:
```suggestion
% FB has been involved in the survey design process, conducted the data analysis, processed the data, and reviewed the written content of the paper. 
```
sez ChatGPT

File: edubots_surveys.Rnw [hunk 2]
```diff
@@ -1789,20 +1787,20 @@ The {\bf first survey}, shown in its original form in Figure \ref{figure:survey1
...
-\item Agenda bots that e.g. remind students about project deadlines
-\item Bots that collect and provide answers to frequent questions
+\item Agenda bots that e.g. remind students project deadlines
+\item Bots that collect and provide answers to frequent asked questions
 \item Bots that inform students about their grades
 \item Bots that help with the class materials (e.g. searching for topics about a concept, or asking about them)
 \end{itemize}
 
-\textit{4. The virtualization of teaching due to the COVID pandemic during this year and the previous one, did it mean any change in your viewpoint or use of messaging apps in class?}
+\textit{4. Regardin the virtualization of teaching due to the COVID-19 pandemic during this year and the previous one, did it mean any change in your viewpoint or use of messaging apps in class?}
```
Comment (JJ) [inline edubots_surveys.Rnw]:
```suggestion
\textit{4. Regarding the virtualization of teaching due to the COVID-19 pandemic during this year and the previous one, did it mean any change in your viewpoint or use of messaging apps in class?}
```

**Review by JJ — APPROVED** [review]
LGTM

File: edubots_surveys.Rnw (hunk 1 above)
Comment (fbarranco) [reply edubots_surveys.Rnw]: Well, he knows better

**Pull request merged**